app/registries/foreman/plugin/rbac_support.rb

```diff
@@ -4,8 +4,10 @@ class RbacSupport
       def add_all_permissions_to_default_roles(all_permissions)
         return if Foreman.in_rake?('db:migrate')
         view_permissions = all_permissions.where("name LIKE :name", :name => "view_%")
+        org_admin_permissions = all_permissions.where("resource_type <> :resource_type", :resource_type => "Organization")
```

Comment:
Nitpick, prefer to use ActiveRecord helpers, they prevent bugs and allow the query to be optimized `where.not(:resource_type => 'Organization')` -